HsYAML.cabal

```
@@ -113,7 +113,7 @@ executable yaml-test
                  , containers
                  , mtl
                    -- non-inherited
-                 , megaparsec  >= 6.5.0 && < 6.6
+                 , megaparsec  >= 7.0.0
```

the upper bound was missed here, e.g.

```suggestion
                 , megaparsec  == 7.0.*
```

---

done.